frontend/src/components/NotFound.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Thanks to Stack Overflow, I found a, easy way to get the custom not found-page to show on Netlify too: https://stackoverflowteams.com/c/technigo/questions/4255